js/apps/admin-ui/src/realm-settings/localization/EffectiveMessageBundles.tsx

```diff
@@ -63,9 +66,12 @@ export const EffectiveMessageBundles = ({
   >({});
   const themes = serverInfo.themes;
   const themeKeys = themes
-    ? Object.keys(themes).sort((a, b) => a.localeCompare(b))
+    ? Object.keys(themes)
+        .filter((key) => key !== "common")
+        .sort((a, b) => a.localeCompare(b))
     : [];
-  const themeTypes = Object.values(themes!)
+
+  const themesList = Object.values(themes!)
     .flatMap((theme) => theme.map((item) => item.name))
     .filter((value, index, self) => self.indexOf(value) === index)
```

Comment:
Why are we filtering out duplicates? Are there cases where there are multiple themes of the same name?

Reply:
Thanks @jonkoops I'm filtering out duplicates because I'm using `  const themes = serverInfo.themes;` which returns this obj:

![Screenshot 2024-01-18 at 13 19 53](https://github.com/keycloak/keycloak/assets/4890675/65afcd0f-5042-4ecf-9b15-9e77f68e1709)

I can just create a hardcoded obj with {base, keycloak, keycloak.v2, keycloak.v3} and not use themes. WDYT?

Reply:
Ahhh, yes this makes sense! Just wanted to understand better, thanks!